images/runtime/dotnetcore/3.1/base.bullseye.Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-trace --version 5.0
 RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-dump --version 5.0.236902
 RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-counters --version 5.0.236902
 RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-gcdump --version 5.0.236902
-RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-monitor --version 6.1.*
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-monitor --version 6.*
```

Comment:
I don't have access to the email chain, but just want to confirm that we do not also want to make this change for [.NET 5](https://github.com/microsoft/Oryx/blob/main/images/runtime/dotnetcore/5.0/base.buster.Dockerfile#L9) as well?

Reply:
also, :shipit:

Reply:
It looks like from the [work item](https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1582251) that it's just .NET 6 and .NET Core 3.1, but good question if we should update .NET 5 as well 🤔

Reply:
Looks like .NET 5 is EOL: 
![image](https://user-images.githubusercontent.com/86327553/186996900-46ee8730-55aa-47af-a6eb-b06f68252ee2.png)

Reply:
I ignored .net 5 because of two reasons:
1) .net 5 was not mentioned anywhere in the story and the email thread.
2) .net 5 is not supported anymore.